src/pages/RecruitList/RecruitList.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import { useInView } from 'react-intersection-observer';
+import * as S from './RecruitListStyled';
+import { BsFillBookmarkFill, BsChevronRight } from 'react-icons/bs';
+import { API } from '../../api';
+export default function RecruitList() {
```

Comment:
```suggestion
import { API } from '../../api';

export default function RecruitList() {
```
이 사이에 한줄 띄어주시면, import 구문이랑 함수 signature를 구분하기 좀더 좋겠습니다

Reply:
넵 적용했습니다